db/migrate/20171126131104_remove_duplicate_taxable_taxonomies.rb

```diff
@@ -1,6 +1,6 @@
 class RemoveDuplicateTaxableTaxonomies < ActiveRecord::Migration[4.2]
   def up
-    deduped = TaxableTaxonomy.group(:taxonomy_id, :taxable_id, :taxable_type).pluck('min(id)')
+    deduped = TaxableTaxonomy.group(:taxonomy_id, :taxable_id, :taxable_type).pluck(TaxableTaxonomy.arel_table[:id].minimum)
```

Comment:
Wondering: doesn't pluck have some syntax that could make this easier? Like `.pluck(:id, :minimum)`. Note this is based just on guessing, not on anything factual.

Reply:
Well, there is this syntax:

```ruby
Person.group(:last_name).having("min(age) > 17").minimum(:age)
```

Reply:
I'm going to use this as an exercise to learn. Is the `having` clause needed? In regular SQL there's isn't AFAIK. Is there an easy way to print the generated SQL query and compare it?

Reply:
@ekohl: You can sometimes use `.to_sql`. It does not work for pluck, there you can substitute `pluck` with `select` to get the query.
If you want to get the full query, you need to turn on SQL logging.